_posts/2019-02-11-lecture-08.md

```diff
@@ -0,0 +1,174 @@
+
+The answer is to switch our choice. If we switch, our probabilty of winning becomes 2/3 rather than the 1/3 if we stick to our choice. We can think of it as follows. There are three options to select from initially, two empty doors and one with the prize money. If we are always switching, we will end up with the prize money if we have initially chosen the empty door and we will lose if we have initially selected the door with the prize money. So to win, we need to select the empty door at first whose probability is 2/3. So the chance of us winning the car if we switch is 2/3.
+
+### Causal Models
+If we have causal models, it is very easy to do intervention. We just need to delete all the incoming edges into the nodes which we are intervening and then do inference on the obtained graphical structure.
+
+Also if we know the causal structure, we can apply some of the known distributions into new scenarios. This idea is used in transfer learning.
+
+### Prediction
+This answers the question of $$ P(Y | X) $$ which means the probability of Y if X happens to take a particular value. This can be found out just from the observational data. We don't need any graphs, or causal realtions to find this value. 
+
+### Intervention
+This answers the question of $$ P(Y | do(X)) $$ which means what is the probability of Y if X is set to a particular value keeping all the other variables same. For this, we definitely need to know all the causal relations of all the variables.
+
+### Counterfactual Thinking
+This answers the question of $$ P(y_x | x', y') $$, which is the probability that event Y = y would be observed had X been x, given that we actually observed X to be $$ x' $$ and and Y to be $$ y' $$. For this, again we require causal relations via the structural equation model.
+
+## Identification of Causal Relation
+
+### Causal DAGs
+Each edge in a causal DAG corresponds to direct causal influence between the nodes and it is asymmetric.
+Let $$ P_x(V) $$ be the distribution resulting from intervention do(X=x). Then a DAG is a causal DAG if
+* $$ P_x(V) $$ is Markov relative to G. This distribution is still factorizable according to the Markov property (product of proabbility of each node given its parents) with respect to G
+    
+* $$ P_x(V_i = v_i) = 1 $$ for $$ V_i \in X $$ and $$ v_i $$ consistent with X=x. This is so because we have performed intervention on X and we have set that to x. Hence we know those variables with absolute certainty.
+    
+* $$ P_x(V_i | PA_i) = P(V_i | PA_i) $$ for all $$ V_i \notin X $$ where $$ PA_i $$ represents the parents of $V_i$. That means the conditional distributions of all the nodes not included in the intervention remains the same before and after the intervention. We can see this graphically since while doing intervention, we just delete all the incoming edges into the nodes on which intervention is performed. So the other nodes remain unchanges(in terms of parents)
```

Comment:
```suggestion
* $$ P_x(V_i \mid PA_i) = P(V_i \mid PA_i) $$ for all $$ V_i \notin X $$ where $$ PA_i $$ represents the parents of $V_i$. That means the conditional distributions of all the nodes not included in the intervention remains the same before and after the intervention. We can see this graphically since while doing intervention, we just delete all the incoming edges into the nodes on which intervention is performed. So the other nodes remain unchanges(in terms of parents)
```

Changing "|" to "\mid" in the LaTeX code seems to fix the equation rendering.